src/test/java/roomescape/integrate/domain/AvailableReservationTimeTest.java

```diff
@@ -0,0 +1,114 @@
+package roomescape.integrate.domain;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import io.restassured.RestAssured;
+import io.restassured.http.ContentType;
+import io.restassured.response.Response;
+import java.time.LocalDate;
+import java.time.LocalTime;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayNameGeneration;
+import org.junit.jupiter.api.DisplayNameGenerator;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.boot.test.context.SpringBootTest.WebEnvironment;
+import org.springframework.jdbc.core.JdbcTemplate;
+import org.springframework.test.annotation.DirtiesContext;
+import org.springframework.test.annotation.DirtiesContext.ClassMode;
+
+@SpringBootTest(webEnvironment = WebEnvironment.DEFINED_PORT)
+@DisplayNameGeneration(DisplayNameGenerator.ReplaceUnderscores.class)
+@DirtiesContext(classMode = ClassMode.BEFORE_EACH_TEST_METHOD)
+class AvailableReservationTimeTest {
+
+    private static String todayDateString;
+
+    @BeforeEach
+    void setup() {
+        todayDateString = LocalDate.now().plusDays(1).toString();
+        Map<String, String> timeParam = new HashMap<>();
+        LocalTime afterTime = LocalTime.now().plusHours(1L);
+        timeParam.put("startAt", afterTime.toString());
+
+        Map<String, String> timeParam2 = new HashMap<>();
+        LocalTime afterTime2 = LocalTime.now().plusHours(2L);
+        timeParam2.put("startAt", afterTime2.toString());
+
+        Map<String, String> timeParam3 = new HashMap<>();
+        LocalTime afterTime3 = LocalTime.now().plusHours(3L);
+        timeParam3.put("startAt", afterTime3.toString());
+
+        Map<String, String> themeParam = new HashMap<>();
+        themeParam.put("name", "테마 명");
+        themeParam.put("description", "description");
+        themeParam.put("thumbnail", "thumbnail");
+
+        Map<String, Object> reservation = new HashMap<>();
+        reservation.put("name", "브라운");
+        reservation.put("date", todayDateString);
+        reservation.put("timeId", 1);
+        reservation.put("themeId", 1);
```

Comment:
```suggestion
        Map<String, Object> reservation = Map.of(
                "name", "브라운",
                "date", todayDateString,
                "timeId", 1,
                "themeId", 1
        );
```
Java 가 제공하는 api 를 적극 활용해보면 좋을 것 같습니다.

Reply:
테스트 전체적으로 적용 완료했습니다!